include/private/common/async_runner.hpp

```diff
@@ -1,133 +0,0 @@
-// Part of measurement-kit <https://measurement-kit.github.io/>.
```

Comment:
File removed because this is superseded by running using `Worker::default_tasks_queue()`.